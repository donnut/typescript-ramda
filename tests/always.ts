import R_always = require('../ramda/dist/src/always');

declare const string: string;

// @dts-jest:pass:snap
R_always(string);
// @dts-jest:pass:snap
R_always(string)();
