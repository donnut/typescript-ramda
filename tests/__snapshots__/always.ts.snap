// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R.always(string) 1`] = `"(...args: any[]) => string"`;

exports[`R.always(string)() 1`] = `"string"`;
